catalog/core/catalog-core-commands/pom.xml

```diff
@@ -129,10 +129,12 @@
                             ddf.catalog.data.impl,
                             ddf.catalog.operation.impl,
                             ddf.catalog.util.impl,
-                            ddf.catalog.filter.impl
+                            ddf.catalog.filter.impl,
+                            ddf.catalog.source.impl
                         </Private-Package>
                         <Export-Package />
                         <Import-Package>
```

Comment:
Any reason we need to explicitly exclude `org.mockito`? It should be in `<scope>test</scope>` and not get pulled in, right?